tests/index/test_index_document_meilisearch.py

```
@@ -555,3 +555,270 @@ def test_update_documents_ndjson(index_with_documents, songs_ndjson):
+    task = index.add_documents(documents)
+    index.wait_for_task(task.task_uid)
+
+    # Now try to update with a new document - should be ignored
+    new_documents = [
+        {"id": "2", "title": "New Document"},
+    ]
+    task = index.update_documents(new_documents, skip_creation=True)
+    index.wait_for_task(task.task_uid)
+
+    # New document should not exist
+    with pytest.raises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)